### PR TITLE
[freshrss] new chart

### PR DESCRIPTION
**Pull request by npawelek**
FreshRSS chart based on the dependent 'common' chart.

Signed-off-by: Nathan Pawelek <nathan@nathanpawelek.com>

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/freshrss/.helmignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+.svn/
+# Common backup files
+*.swp
+*.bak
+*.tmp
+*~
+# Various IDEs
+.project
+.idea/
+*.tmproj
```
Comment (onedr0p) [inline charts/freshrss/.helmignore:21]: need to add in .vscode/

File: charts/freshrss/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+sources:
+  - https://github.com/FreshRSS/FreshRSS
+  - https://hub.docker.com/r/linuxserver/freshrss
+maintainers:
+  - name: npawelek
+    email: nathan@nathanpawelek.com
+dependencies:
+  - name: common
+    repository: https://k8s-at-home.com/charts/
+    version: ^1.6.1
```
Comment (onedr0p) [inline charts/freshrss/Chart.yaml:20]: change this version to `1.6.1` (without the carrot)

**Review by onedr0p — CHANGES_REQUESTED** [review]
Few changes but over awesome, and thanks for using the common library!

File: charts/freshrss/Chart.yaml (hunk 2 above)
Comment (onedr0p) [reply charts/freshrss/Chart.yaml:20]: there's a huge PR I submitted to change this in the other charts. this is to prepare for renovatebot we will be adding in sometime soon to help with dependencies :)

File: charts/freshrss/.helmignore (hunk 1 above)
Comment (npawelek) [reply charts/freshrss/.helmignore:22]: Done!

File: charts/freshrss/Chart.yaml (hunk 2 above)
Comment (npawelek) [reply charts/freshrss/Chart.yaml]: Done!

**Comment by onedr0p** [comment]
Gonna close and reopen to kick off the CI again, that error is known and hopefully we will fix it soon

**Review by onedr0p — APPROVED** [review]

**Pull request merged**